dataprofiler/profilers/profiler_options.py

```diff
@@ -269,6 +269,42 @@ def _validate_helper(self, variable_path: str = "ModeOption") -> list[str]:
         return errors
 
 
+class NumQuantilesOption(BooleanOption):
```

Comment:
This class will need its own tests Class as well. maybe mimic similar to PrecisionOptions:
https://github.com/capitalone/DataProfiler/blob/main/dataprofiler/tests/profilers/profiler_options/test_precision_options.py